openam-schema/openam-saml2-schema/pom.xml

```diff
@@ -83,12 +84,7 @@
 
         <dependency>
             <groupId>com.sun.xml.bind</groupId>
-            <artifactId>jaxb-libs</artifactId>
-        </dependency>
-
-        <dependency>
-            <groupId>com.sun.xml.bind</groupId>
-            <artifactId>jaxb-xjc</artifactId>
+            <artifactId>jaxb-impl</artifactId>
```

Comment:
Is this intentional? XJC is not the same as IMPL.

Reply:
XJC is not required since the generated code was commited into the repository and the jaxb2-maven-plugin was commented in the past.

Anyway this solution is ugly a we should let the code to be generated by the plugin again.

Reply:
I will create an issue to solve this.